src/main/java/com/teamsfinder/userwriteservice/user/model/Tag.java

```diff
@@ -0,0 +1,17 @@
+package com.teamsfinder.userwriteservice.user.model;
```

Comment:
tagi będą wykorzystywane nie tylko w userze więc ten package to średnio

Reply:
To jak to pakietowac?

Reply:
com.teamsfinder.userwriteservice.tag.model; normalnie